File: C++/001_Two_Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res(2) ;
        for ( int i = 0 ; i < nums.size() ; i++ ) {
            for ( int j = i + 1 ; j < nums.size() ; j ++ ){
                if ( nums[i] + nums[j] == target) {
                    res[0] = i ;
                    res[1] = j ;
                    return res ;
                }
            }
        }
    }
};

File: Python/001_Two_Sum.py
class Solution:
    def twoSum(self, nums, target):
        for i , item in enumerate(nums) :
            temp = target - item
            if nums.count(temp) != 0 :
                j = nums.index(temp)
                if i != j :
                    res = []
                    res.append(i)
                    res.append(j)
                    return res
